### DISCUSSION TITLE
Struggling to install the app to contribute

### DISCUSSION THREAD

**Discussion by ahmed-n-abdeltwab** [lnreader/lnreader#508] [Q&A] [answered]
Hello, my name is Ahmed Nasser, and I'm an Egyptian software developer.
I'd want to start my open-source journey with this excellent project, **lnreader**.
However, I've been having trouble installing the app; I followed the contribution guide, but the program keeps crashing.
The NPM crashed when I performed NPM `install`; I had to use the **force** flag to get rid of the **issue**, but the problem is that the app crashes with every NPM command I execute; I tried `NPM run start`, `NPM run lint`, and `NPM run android`, but all of them failed.
The issue is that I want to help, but I can't debug or even execute the program.

**Comment by rajarsheechatterjee** [comment, marked as answer]
Could you share the error you are getting while running the commands? 

You can join the app's discord server for getting help with the setup.

**Reply by ahmed-n-abdeltwab** [reply]
Okay, I'll join in.

**Reply by ahmed-n-abdeltwab** [reply]
I'll read the message on Discord, and if I don't find anyone else with the same issue, I'll ask.
